DSESwift/DataPoint.swift

```diff
@@ -55,7 +55,7 @@ public class DataPoint{
      * Set time of the data point.
      * @param time: NSDate for the timestamp of the datapoint.
      **/
-    func setTime(time: NSDate){
+    public func setTimeWithNSDate(time: NSDate){
```

Comment:
@kanekin this feel soo objecttive-c. is it still a thing in swift? if not, i think it's good time to stop using it. even if it's a must because of obj-c integration, maybe we can make it like a proxy function